src/ansys/simai/core/data/models.py

```diff
@@ -38,6 +114,43 @@ class ModelConfiguration:
     simulation_volume: Dict[str, Any] = None
     project_id: str = None
 
+    @property
+    def domain_of_analysis(self) -> DomainOfAnalysis | None:
```

Comment:
```suggestion
    simulation_volume: Dict[str, Any] = None
    project_id: str = None

    @property
    def domain_of_analysis(self) -> Optional[DomainOfAnalysis]
```
The `|` operator was added in 3.10 and we support from 3.9. The worrying part is that the CI passed.

Reply:
merge operator removed.